miniapp/USERGUIDE.md

```diff
@@ -1,7 +1,3 @@
----
-layout: userguide
```

Comment:
This is needed for the github pages theme. See here: https://rakutentech.github.io/android-miniapp/

We direct people to the github pages site for documentation rather than the markdown file.

Reply:
Thanks for sharing the context. I'll close this PR for now.